pkgs/by-name/ma/maltego/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, runtimeShell
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
+    url = "https://downloads.maltego.com/maltego-v4/linux/Maltego.v${version}.linux.zip";
+    hash = "sha256-q+1RYToZtBxAIDSiUWf3i/3GBBDwh6NWteHiK4VM1HY=";
+  };
+
+  preConfigure = ''
+      for awks in bin/maltego; do
+          substituteInPlace $awks \
+                --replace /usr/bin/awk ${gawk}/bin/awk
+      done
+  '';
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  buildInputs = [ jre giflib ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share}
+    chmod +x bin/${pname}
```

Comment:
Please do not overuse pname

Reply:
Replaced `${pname}` to `maltego` in `chmod`.